src/uu/df/df.rs

```diff
@@ -0,0 +1,878 @@
+    ($str: expr) => {
+        OsString::from($str.clone())
+            .as_os_str()
+            .encode_wide()
+            .chain(Some(0))
+            .collect::<Vec<u16>>()
+            .as_ptr()
+    };
+}
+
+#[cfg(windows)]
+#[allow(non_snake_case)]
+fn LPWSTR2String(buf: &[u16]) -> String {
+    let len = unsafe { libc::wcslen(buf.as_ptr()) };
+    OsString::from_wide(&buf[..len as usize])
+        .into_string()
+        .unwrap()
+}
+
+fn get_usage() -> String {
+    format!("{0} [OPTION]... [FILE]...", executable!())
+}
+
+#[cfg(any(target_os = "freebsd", target_os = "macos"))]
+extern "C" {
+    #[cfg(all(target_os = "macos", target_arch = "x86_64"))]
+    #[link_name = "getmntinfo$INODE64"]
+    fn getmntinfo(mntbufp: *mut *mut statfs, flags: c_int) -> c_int;
+
+    #[cfg(all(
+        target_os = "freebsd",
+        not(all(target_os = "macos", target_arch = "x86_64"))
+    ))]
+    fn getmntinfo(mntbufp: *mut *mut statfs, flags: c_int) -> c_int;
+}
+
+#[cfg(any(target_os = "freebsd", target_os = "macos"))]
+impl From<statfs> for MountInfo {
+    fn from(statfs: statfs) -> Self {
+        let mut info = MountInfo {
+            dev_id: "".to_string(),
+            dev_name: unsafe {
+                CStr::from_ptr(&statfs.f_mntfromname[0])
+                    .to_string_lossy()
+                    .into_owned()
+            },
+            fs_type: unsafe {
+                CStr::from_ptr(&statfs.f_fstypename[0])
+                    .to_string_lossy()
+                    .into_owned()
+            },
+            mount_dir: unsafe {
+                CStr::from_ptr(&statfs.f_mntonname[0])
+                    .to_string_lossy()
+                    .into_owned()
+            },
+            mount_root: "".to_string(),
+            mount_option: "".to_string(),
+            remote: false,
+            dummy: false,
+        };
+        info.set_missing_fields();
+        info
+    }
+}
+
+impl FsSelector {
+    fn new() -> FsSelector {
+        FsSelector {
+            include: HashSet::new(),
+            exclude: HashSet::new(),
+        }
+    }
+
+    #[inline(always)]
+    fn include(&mut self, fs_type: String) {
+        self.include.insert(fs_type);
+    }
+
+    #[inline(always)]
+    fn exclude(&mut self, fs_type: String) {
+        self.exclude.insert(fs_type);
+    }
+
+    fn should_select(&self, fs_type: &str) -> bool {
+        if self.exclude.contains(fs_type) {
+            return false;
+        }
+        self.include.len() <= 0 || self.include.contains(fs_type)
+    }
+}
+
+impl Options {
+    fn new() -> Options {
+        Options {
+            show_local_fs: false,
+            show_all_fs: false,
+            show_listed_fs: false,
+            show_fs_type: false,
+            show_inode_instead: false,
+            print_grand_total: false,
+            block_size: match env::var("BLOCKSIZE") {
+                Ok(size) => size.parse().unwrap(),
+                Err(_) => 512,
+            },
```

Comment:
```suggestion
            // block_size: match env::var("BLOCKSIZE") {
            //    Ok(size) => size.parse().unwrap(),
            //    Err(_) => 512,
            //},
```
`cargo clippy` says 'block_size' isn't used; changed to comment if you think it might be used later.